SPEC.md

```diff
@@ -5053,19 +5057,21 @@ workflow name {
 
 ### Workflow Elements
 
-Tasks and workflows have several elements in common. These sections have nearly the same usage in workflows as they do in tasks, so we just link to their earlier descriptions.
+Tasks and workflows have several elements in common. When applicable, the task definition for these sections is linked to rather than duplicated. 
```

Comment:
I felt the wording for this section was a bit too verbose. This wording simplifies things IMO:

- Here is a list of sections, as with tasks.
- We state much more simply that workflows have common elements with tasks and that we will link back to them.